solana-programs/anchor/audius-data/tests/follows.ts

```diff
@@ -222,8 +225,64 @@ describe("follows", function () {
       expect(decodedData.followeeHandle.seed).to.deep.equal(
         constants2.handleBytesArray
       );
-      expect(accountPubKeys[0]).to.equal(adminStorageKeypair.publicKey.toString());
-      expect(accountPubKeys[3]).to.equal(newUser1Key.publicKey.toString());
+      expect(accountPubKeys[0]).to.equal(
+        adminStorageKeypair.publicKey.toString()
+      );
+      expect(accountPubKeys[5]).to.equal(newUser1Key.publicKey.toString());
```

Comment:
nit, perhaps out of scope of this pr but we should possibly destructure the `accountPubKeys` returned to named vars that point to what they're supposed to be - as a person reading the test it seems like magic to assert that a random index is equal to pubkey of the user.

Reply:
yes, if we can do this using the instruction decoder that is absolutely the best way but is also out of scope for this PR